TMS.Net07.Homework.ConverterCalculator/TMS.Net07.Homework.ConverterCalculator.Easy/TMS.Net07.Homework.ConverterCalculator/Program.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.Net07.Homework.ConverterCalculator
+{
+    class Program
+    {
+
+        enum ExchangeSellCourses 
+        {
+            USD_to_BYN = 26590,
+            EUR_to_BYN = 31630,
+            RUB_to_BYN = 342,
+
+            EUR_to_USD = 12120,
+            EUR_to_RUB = 932300,
+            USD_to_RUB = 774700,
+        }
+
+        enum ExchangeBuyCourses
+        {
+            BYN_to_USD = 26240,
+            BYN_to_EUR = 31960,
+            BYN_TO_RUB = 352,
+
+            USD_to_EUR = 11940,
+            RUB_to_EUR = 907600,
+            RUB_to_USD = 754200
+        }
+
+        static void Main(string[] args)
+        {
+            ExchangeSellCourses SellCourse;
+            ExchangeBuyCourses BuyCourse;
+
+            string sourceCurrency;
+            string targetCurrency;
+            string amountInput;
+
+            bool sellCheck;
+            bool buyCheck;
+            bool amountCheck;
+
+            double amount;
+            double AmountAfterExchange;
```

Comment:
Names should be consistent. Please, use `camelCase` for all variables
```suggestion
            double amountAfterExchange;
```

Reply:
You mean lowerCamelCase?

Reply:
Exactly ;)